source/manual/move-apps-between-servers.html.md

```diff
@@ -0,0 +1,103 @@
+
+## Carrenza
+
+### Create the new servers (if required)
+
+If you're moving an app to new servers, start by creating those server.
+
+1. Add [vCloud configuration][govuk-provisioning] for production and staging (you can choose any IP addresses that are not currently used as long as they use the same prefix as others in the same file).
+1. Add the IP addresses for the new servers and a new Puppet node class in [govuk-puppet].
+1. Deploy puppet in staging and production.
+1. In staging, SSH to the puppetmaster and run a loop to sign SSL certificates for the new servers as they're created:
+    ```bash
+    $ ssh puppetmaster-1.management.staging
+    $ while true; do sudo puppet cert sign --all; sleep 10; done
+    ```
+1. Run the [Launch VMs][launch-vms-jenkins] Jenkins job, using the [Carrenza staging username and password][carrenza-credentials], to create the new servers.
+1. Once the job has completed, terminate the loop on the puppetmaster.
+1. Re-run the loop and Jenkins job in production.
+
+[govuk-provisioning]: https://github.com/alphagov/govuk-provisioning/pull/41
+[govuk-puppet]: https://github.com/alphagov/govuk-puppet/pull/7294
+[launch-vms-jenkins]: https://deploy.staging.publishing.service.gov.uk/job/Launch_VMs/
+[carrenza-credentials]: https://github.com/alphagov/govuk-secrets/blob/master/pass/2ndline/carrenza/vcloud-staging.gpg
+
+### Add the relevant app(s) to the new servers
+
+Once the servers are created, they will run puppet to apply relevant configuration, but there will be no apps.
+
+1. Change the [hieradata][] to add the relevant app(s) to the new servers.
+1. Deploy puppet in staging and production.
+1. To speed up the process, you can run puppet manually on the new servers to deploy the apps.
+1. Change the relevant [app deployment scripts][deploy-scripts] to deploy to both the old and new servers.
+
+[hieradata]: https://github.com/alphagov/govuk-puppet/pull/7302
+[deploy-scripts]: https://github.com/alphagov/govuk-app-deployment/pull/247
+
+### Add the new servers to the load balancers
+
+Once you've verified that the app(s) have been deployed to all the new servers, you'll need to change the load balancers to start using the new servers as part of a managed migration from the old to the new servers.
+
+1. Change the [hieradata] for the load balancers to balance between both the old and new servers. For this step, it is important that all servers are running the same version of the app, so if appropriate, re-deploy the app now.
+1. Deploy puppet in staging and production.
+
+The app will now be running on both old and new servers and the load balancers will use both as their configuration is updated by puppet.
+
+[hieradata]: https://github.com/alphagov/govuk-puppet/pull/7285
+
+### Remove the old servers from the load balancers
+
+Once all the load balancers have been updated, requests for the app(s) will be using both sets of servers, which will allow you to remove the old servers from serving the app(s).
```

Comment:
maybe "requests to the app(s)" instead of "requests for the app(s)"

Reply:
Fixed